site/src/pages/ManagementSettingsPage/SidebarView.tsx

```diff
@@ -396,7 +412,7 @@ const classNames = {
   `,
 
 	subLink: (css, theme) => css`
-    color: inherit;
+    color: ${theme.palette.text.secondary};
```

Comment:
De-emphasized the colors of the inactive sub links because in dark mode, there wasn't enough contrast to make it obvious which subheader was active